A mass change from http://xbmc.org to http://kodi.tv.

kekePower opened the pull request:
Change http://xbmc.org to http://kodi.tv

## Description
I used the command
grep --include={\*.cpp,\*.h} -rnl './' -e "http://xbmc.org" | xargs -i@ sed -i 's/http\:\/\/xbmc\.org/http\:\/\/kodi\.tv/g' @
to change every http://xbmc.org to http://kodi.tv in *.cpp and *.h files

## Motivation and Context
I read in a PR once that it was a wish to switch all references to xbmc.org over to kodi.tv.
When I found a great grep/sed command that did the trick in one go, I decided I wanted to give it a try.

A lot of files has been touched, so if you want to reject this PR, I'm fine with it.

## How Has This Been Tested?
No need to test as this only touches references to http://xbmc.org.

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

kekePower committed:
A mass change from xbmc.org to kodi.tv.

I used the command
grep --include={\*.cpp,\*.h} -rnl './' -e "http://xbmc.org" | xargs -i@ sed -i 's/http\:\/\/xbmc\.org/http\:\/\/kodi\.tv/g' @
to change every http://xbmc.org to http://kodi.tv in *.cpp and *.h files

kekePower committed:
A mass change from xbmc.org to kodi.tv.

I used the command
grep --include={\*.cpp,\*.h} -rnl './' -e "http://xbmc.org" | xargs -i@ sed -i 's/http\:\/\/xbmc\.org/http\:\/\/kodi\.tv/g' @
to change every http://xbmc.org to http://kodi.tv in *.cpp and *.h files

Fixed issues mentioned by @notspiff
URL in tools/depends/native/JsonSchemaBuilder/src/JsonSchemaBuilder.cpp
Windows EOL in
xbmc/platform/Filesystem.h
xbmc/platform/win32/Filesystem.cpp
xbmc/utils/Literals.h

notspiff commented on tools/depends/native/JsonSchemaBuilder/src/JsonSchemaBuilder.cpp:
```diff
@@ -153,7 +153,7 @@ int main(int argc, char* argv[])
   out << "namespace JSONRPC" << endl;
   out << "{" << endl;
-  out << "  const char* const JSONRPC_SERVICE_ID          = \"http://xbmc.org/jsonrpc/ServiceDescription.json\";" << endl;
+  out << "  const char* const JSONRPC_SERVICE_ID          = \"http://kodi.tv/jsonrpc/ServiceDescription.json\";" << endl;
```
remove this change please. affects RPC, needs to verify it won't break things and don't want to apply brain in this pr. you can open a separate pr for it if you cba

notspiff commented on xbmc/platform/Filesystem.h:
```diff
@@ -1,7 +1,7 @@
 /*
 *      Copyright (C) 2005-2017 Team Kodi
-*      http://xbmc.org
+*      http://kodi.tv
```
a windows eol snuck in.

notspiff commented on xbmc/platform/win32/Filesystem.cpp:
```diff
@@ -1,6 +1,6 @@
 /*
  *      Copyright (C) 2005-2017 Team Kodi
- *      http://xbmc.org
+ *      http://kodi.tv
```
windows eol (nope, not okay in windows files either).

notspiff commented on xbmc/utils/Literals.h:
```diff
@@ -1,7 +1,7 @@
 /*
  *      Copyright (C) 2014 Team XBMC
- *      http://xbmc.org
+ *      http://kodi.tv
```
windows eol

notspiff reviewed (changes requested):
RPC change is hard, the others i realize are not your doing but if cba to address them while at it, you get an extra cookie.

kekePower committed:
Fixed issues mentioned by @notspiff
URL in tools/depends/native/JsonSchemaBuilder/src/JsonSchemaBuilder.cpp
Windows EOL in
xbmc/platform/Filesystem.h
xbmc/platform/win32/Filesystem.cpp
xbmc/utils/Literals.h

kekePower commented:
@notspiff 
I do like cookies :-)

notspiff commented:
now to get sugar coating; you should put the json part of the second commit into the first. you know your way around git squashery ?

kekePower commented:
@notspiff 
Do you want only the json part or is it OK to sqash the whole second commit?

Rechi commented:
wait for #13427 getting merged and then rebase onto master again (and squash to a single commit)

kekePower commented:
@Rechi Ok, will wait.

Rechi committed:
convert CRLF to LF line endings

Rechi committed:
Merge pull request #13427 from Rechi/convertLineEndings

convert CRLF to LF line endings

kekePower committed:
Rebased patch after #13427

kekePower commented:
@notspiff 
No, I guess I really don't know anything about git squash even though I tried to read up on it.

notspiff commented:
okay, then i will handle that part. no worries.

kekePower commented:
@notspiff Tusen takk :-)

notspiff commented:
bare hyggelig.

pushed manually in rebased  form. thanks for your contribution!

da-anda commented:
while at it, I guess we should also replace `Team XBMC` with `Team Kodi` :)
